bin/propolis-server/src/lib/stats/pvpanic.rs

```diff
@@ -26,11 +26,19 @@ pub struct PvpanicProducer {
 
 /// An Oximeter `Metric` that specifies the number of times an instance's guest
 /// reported a guest-handled kernel panic using the QEMU `pvpanic` device.
+//
+// NOTE: We may want to collapse these into a single metric with a field
+// indicating whether the guest or host handle bit was set, but it's not clear
+// that these are truly mutually exclusive. It could also be done with two
+// boolean fields, which would allow either to be set.
```

Comment:
File a ticket about this?

Reply:
https://github.com/oxidecomputer/propolis/issues/660